imports/api/users/user.methods.js

```diff
@@ -16,6 +16,10 @@ Meteor.methods({
     try {
       const userId = Accounts.createUser(values);
       if (userId) {
+        const accountVerification = Accounts.sendVerificationEmail(userId);
```

Comment:
This is causing the email to be sent already once. And then in the following method `sendVerificationLink`, there's another email coming. So there are two emails that arrive...